perllib/FixMyStreet/App/Controller/Root.pm

```diff
@@ -77,6 +77,10 @@ sub index : Path : Args(0) {
         $c->detach;
     }
 
+    # TODO: Not sure we want to hammer the FS for every front page request,
```

Comment:
I agree, this will hurt performance. Could the stash function check/set something in Memcache, and then an edit/delete of the icons clears it much like it already does for States?

Reply:
Good idea, will add that.